### PR TITLE
Add extra words to Arabic provider [Address, Company, Internet]

### PR DESCRIPTION
**Pull request by MoaazGouda**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 vendor
 composer.lock
+nbproject
```
Comment (GrahamCampbell) [inline .gitignore]: no

Comment (MoaazGouda) [inline .gitignore]: removed

File: src/Faker/Provider/ar_SA/Internet.php [hunk 2]
```diff
@@ -11,18 +11,22 @@ class Internet extends \Faker\Provider\Internet
...
     );
 
     protected static $tld = array(
-        'biz', 'com', 'info', 'sa', 'net', 'org',
+        'biz', 'com', 'info', 'sa', 'net', 'org', 'com.eg',
     );
 
     protected static $lastNameAscii = array(
-        'abbad', 'abbadi', 'abbas', 'abulebbeh', 'flefel', 'hadi', 'hamad', 'hasan', 'jabri', 'kanaan', 'karam', 'maanee', 'melhem', 'nimry', 'obaisi', 'qasem', 'qawasmee', 'rabee', 'rashwani', 'shami', 'zaloum',
+        'abbad', 'abbadi', 'abbas', 'abulebbeh', 'flefel', 'hadi', 'hamad', 'hasan', 'jabri', 'kanaan', 'karam', 'maanee', 'melhem',
```
Comment (fzaninotto) [inline src/Faker/Provider/ar_SA/Internet.php]: Where does this list come from? Please quote the source in the comments. Same for the first names.

**Comment by fzaninotto** [comment]
Each time you add or remove one element in a Faker dataset, every test in this locale using seeding start to fail. That means that we must restrict the occurrences of data updates to major ones. Your change is relatively minor. Please provide more data, or I'll have to close this PR.

File: src/Faker/Provider/ar_SA/Internet.php (hunk 2 above)
Comment (MoaazGouda) [inline src/Faker/Provider/ar_SA/Internet.php]: You mean where geographically these names are known?

**Comment by MoaazGouda** [comment]
@fzaninotto Francois, i ran the unit test and i see no errors, just skipped tests; could you please inform me which test fails?
also i found too many errors/warnings while running phpcs coming from other parts which haven't any changes by me. was it merged with these errors exist?

File: src/Faker/Provider/ar_SA/Internet.php (hunk 2 above)
Comment (fzaninotto) [inline src/Faker/Provider/ar_SA/Internet.php]: No, the source for this data. The website / encyclodepia / database

**Comment by fzaninotto** [comment]
At the bottom of this PR is a big red cross saying "all checks have failed". Click on the "details" link of the Travis CI build to see the errors.

**Comment by MoaazGouda** [comment]
@fzaninotto Ok, will close this PR, and will request another one fixed.

**Pull request closed without merging**